src/main/java/cz/student/adambrg/TaskManager.java

```diff
@@ -1,9 +1,146 @@
 package cz.student.adambrg;
 
-public class TaskManager {
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.io.PrintWriter;
+import java.io.IOException;
+import java.time.format.DateTimeFormatter;
 
-    public static void main(final String[] args) {
+import org.apache.commons.lang3.ArrayUtils;
 
+public class Main {
+    static final String fileName = "tasks.csv";  //soubor, ze ktereho ziskavame informace
+    static final String[] OPTIONS = {"add", "remove", "list", "exit"};
+    static String[][] tasks; // pole, do ktereho se zapisuji informace ze souboru
+    static Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) { //hlavni metoda, ktera načte data ze souboru a zavola "setOptions" a spusti tim program
+        retrieveData();
+        setOptions();
+    }
+
+    public static void displayOptions() { //metoda, ktera nam vypise vsechny moznosti
+        System.out.println(ConsoleColors.BLUE + "Please select an option: ");
+        for (String option : OPTIONS) {
+            System.out.println(ConsoleColors.RESET + option);
+        }
+    }
+
+    public static void setOptions() { //metoda, ktera umoznuje uzivateli vybrat vystup
+        while (true) {
+            displayOptions();
+            String input = scanner.nextLine();
+            switch (input.toLowerCase()) {
```

Comment:
CLEAN-CODE: promenna input se pouziva pouze ve switch, proto bych nacteni dat dal primo do switche.
```suggestion
            switch ( scanner.nextLine().toLowerCase()) {
```

Reply:
opraveno